string map_name
string[] pose_names
string[] pose_ids
